tile_fetch.py

```diff
@@ -168,7 +168,13 @@ def main():
             except (ValueError, AssertionError):
                 print("Not a valid zoom level.")
 
-    coro = load_tiles(image_info, zoom, args.outfile)
+    try:
+        comp_quality = int(input("What image quality would you like? (0-95, higher is better): "))
+        assert 0 <= comp_quality <= 95
+    except (ValueError, AssertionError):
+        print("Not a valid quality of compression. Please select a number from 0-95")
+
+    coro = load_tiles(image_info, zoom, args.outfile, comp_quality)
```

Comment:
I think it would be better to have the quality as a command-line argument than to ask it dynamically. 
It would make it easier to automate the extraction of several images.

Also, on input errors, your current code prints a message but keeps the invalid `comp_quality`.

Reply:
Sure! I hope the commit I just pushed (https://github.com/kolt54321/gapdecoder/commit/8f1519637f4d842ba90085dbd49090eae2f8fece) solved the change (good idea about changing it to a command-line argument). Programming and programming are still really new for me, so thanks for walking me through this.